paasta_tools/mesos_maintenance.py

```diff
@@ -200,6 +200,49 @@ def is_host_down(hostname=getfqdn()):
     return hostname in get_down_hosts()
 
 
+def get_hosts_forgotten_draining():
+    """Find hosts that are still marked as draining (rather than down) after the start
+    of their maintenance window. For each of these hosts, either mark them as 'down' for
+    maintenance, adjust the time of their maintenance window, or cancel the maintenance
+    window if it is no longer needed.
+    :returns: a list of hostnames of hosts forgotten draining
```

Comment:
These docs aren't accurate (right?)- can you remove the ``For each of these hosts, either mark them as 'down' for maintenance, adjust the time of their maintenance window, or cancel the maintenance window if it is no longer needed.`` please?